static/docs/command-reference/checkout.md

```diff
@@ -94,6 +81,19 @@ be pulled from remote storage using `dvc pull`.
   remove files that don't match those DVC-file references or are missing from
   cache. (They are not "committed", in DVC terms.)
 
+- `-d`, `--with-deps` - determine files to update by tracking dependencies to
+  the target DVC-files (stages). This option only has effect when one or more
+  `targets` are specified. By traversing all stage dependencies, DVC searches
+  backward from the target stages in the corresponding pipelines. This means DVC
+  will not checkout files referenced in later stages than the `targets`.
+
+- `--relink` - ensures the file linking strategy (`reflink`, `hardlink`,
+  `symlink`, or `copy`) for all data files in the workspace is consistent with
+  the project's [`cache.type`](/doc/command-reference/config#cache). This is
+  achieved by recreating **all <abbr>outputs</abbr>** referenced in current
```

Comment:
recreating - sounds like we run `repro`

outputs - again, not sure we need it here

Reply:
Replaced "recreating" for "restoring" in 1813f942. "outputs" I removed previously from here.